src/TurbKineticEnergyEquationSystem.C

```diff
@@ -337,9 +340,15 @@ TurbKineticEnergyEquationSystem::register_interior_algorithm(
         case SST_TAMS:
           nodeAlg.add_kernel<TKESSTTAMSNodeKernel>(realm_.meta_data(), realm_.solutionOptions_->get_coordinates_name());
           break;
+        case SST_IDDES:
+            nodeAlg.add_kernel<TKESSTIDDESNodeKernel>(realm_.meta_data());
+            break;
+        case SST_IDDES_ABL:
+            nodeAlg.add_kernel<TKESSTIDDESABLNodeKernel>(realm_.meta_data());
+            break;
         default:
           std::runtime_error("TKEEqSys: Invalid turbulence model, only SST, "
-                             "SST_DES, SST_TAMS and  Ksgs supported");
+                             "SST_DES, SST_TAMS, SST_IDDES, SST_IDDES_ABL and  Ksgs supported");
```

Comment:
Looks like this will never happen as the constructor already throws

Reply:
Would your rather remove the default switch statement altogether?  I kind of like leaving it in, but I'd change the error message to not list the models since it should serve as a developer warning.  I'm fine removing it altogether if you'd prefer though.

Reply:
I don't think we can remove the `default` because it would generate compile time warnings. How about adding just a comment so that developers know what is going on? 

```C++
default:
  // Should never reach here as we check and throw in constructor
  break;
```

Reply:
Sure that works for me.